### PR TITLE
Update conferences.yml

### PR DESCRIPTION
**Pull request by Amir-Arsalan**

### CODE REVIEW

File: _data/conferences.yml [hunk 1]
```diff
@@ -72,13 +72,13 @@
...
 
 - name: CVPR
-  year: 2018
-  id: cvpr18
-  link: http://cvpr2018.thecvf.com/
-  deadline: "2017-11-15 23:59:59"
+  year: 2019
+  id: cvpr19
+  link: http://cvpr2019.thecvf.com/
+  deadline: "2017-11-16 23:59:59"
```
Comment (abhshkdz) [inline _data/conferences.yml]: should be 2018-11-16, right?

Comment (Amir-Arsalan) [reply _data/conferences.yml]: Yes. Fixed now

File: _data/conferences.yml [hunk 2]
```diff
@@ -75,7 +75,7 @@
   year: 2019
   id: cvpr19
   link: http://cvpr2019.thecvf.com/
-  deadline: "2017-11-16 23:59:59"
+  deadline: "2019-11-16 23:59:59"
```
Comment (abhshkdz) [inline _data/conferences.yml]: It should be **2018-11-16**, not 2019.

Comment (Amir-Arsalan) [reply _data/conferences.yml]: Sorry, my bad. Fixed now .

**Comment by abhshkdz** [comment]
Thanks @Amir-Arsalan!

**Pull request merged**